src/tests-tools/test.utils.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { ReactElement } from 'react'
+import { render, RenderOptions } from '@testing-library/react'
+import { StyleProvider } from '@ledgerhq/react-ui'
+import userEvent from '@testing-library/user-event'
+
+type PropsTheme = {
+	children: React.ReactNode
+	theme?: 'dark' | 'light'
+}
+
+const AllProviders = ({ children, theme = 'dark' }: PropsTheme) => {
+	userEvent.setup()
+	return (
+		<StyleProvider selectedPalette={theme} fontsPath="/fonts">
+			{children}
+		</StyleProvider>
+	)
+}
+
+const customRender = (
+	ui: ReactElement,
+	options?: Omit<RenderOptions, 'wrapper'>,
+) => render(ui, { wrapper: AllProviders, ...options })
```

Comment:
Could you return the result of userEvent.setup() as done in the documentation, please?
https://testing-library.com/docs/user-event/intro 
That way we will reuse the same instance and will not need to import userEvent in each test

Reply:
Yes of course, doing that right now !